### ISSUE TITLE
Auto restocking

### ISSUE THREAD

**Issue by 2ndHouse** [Sonaza/Vendorer#5]
Wondering if you might add an auto restocking feature similar to what this addon does...

https://mods.curse.com/addons/wow/vendomatic

Seems like something that might work well in Vendorer. Maybe could add the option to the shift click item menu that  you already introduced?

**Comment by Sonaza** [comment]
Possible but I'm just wondering what is the use case for this kind of feature nowadays. All spell cast reagents have been removed and there isn't much to restock. Would it be used for crafting reagents?

**Comment by 2ndHouse** [comment]
Yeah, I guess it's not as handy of a feature as it used to be. Vendor wise, I mostly use it to auto stock water and food.

I used to use an addon called StealYourCarbon that also restocked from your bank also, which was handy for crafting, but the addon broke with 7.0.
